Add optional event type filter to specs (#6)
vippsas/feedapi-spec: SPEC.md

  more events than requested. If not specified,
  an implementation-dependent default is used.

`event-types` (optional): If specified, indicate the subset of event types that
  should be returned by the stream. Any events that does not match will be
  skipped. Types should be delimited by ';' and implementations should be case

>   insensitive.

Author: Add: 'This parameter is optional to support by the server' @Dmytro?

  more events than requested. If not specified,
  an implementation-dependent default is used.

> `event-types` (optional): If specified, indicate the subset of event types that
>   should be returned by the stream. Any events that does not match will be
>   skipped. Types should be delimited by ';' and implementations should be case
>   insensitive.

Reviewer: ```suggestion
`event-types` (optional$^{*}$): If specified, indicates the subset of event types that
  should be returned by the stream. Any events that do not match will be
  skipped. Types should be delimited by `;` and implementations should be case-insensitive.
```

  ```http
  GET https://service/myfeed/events?event-types=ewallet.preparedepositrequested;ewallet.confirmdepositrequested&token=xaf32&partition=16000&cursor=f1ceaa92eb7c11eda43d6fb319691265
  ```

>   \* Optional parameters are optional, not only for the clients, but also for

Reviewer: ```suggestion
$^{*}$ Optional parameters are optional, not only for the clients, but also for
```
